site/content/blogs/apprunner-waf.en.md

```diff
@@ -0,0 +1,141 @@
+If you don’t already have a Request-Driven Web Service, 
+you can run the following command to create and configure an App Runner service.
+```console
+copilot init \
+  --svc-type "Request-Driven Web Service" \
+  --name "waf-example" \
+  --dockerfile path/to/Dockerfile
+```
+Alternatively, you can simply run `copilot svc init` without any flags. Copilot will prompt for information and
+guide you through the process.
+
+### Step 2 (Optional): Create an `addons/` folder for your service
+
+Stay in the workspace where your Request-Driven Web Service is. If you followed step 1, then your working directory's 
+structure may look like:
+```term
+.
+└── copilot/
+  └── waf-example/ # The name of your Request-Driven Web Service. Not necessarily "waf-example".
+      └── manifest.yml
+```
+
+If you don't have `addons/` folder under `./copilot/<name of your Request-Driven Web Service>` yet, create one.
+Now your workspace may look like:
+```term
+.
+└── copilot/
+  └── waf-example/ # The name of your Request-Driven Web Service. Not necessarily "waf-example".
+      ├── manifest.yml
+      └── addons/
+```
+
+### Step 3: Associate Web ACL with your Request-Driven Web Service using addon.
+
+In the addons folder, create two new files: `waf.yml` and `addons.parameters.yml`. Your folders would now look like this:  
+
+  ```term
+  .
+  └── copilot
+      └── waf-example/ # The name of your Request-Driven Web Service. Not necessarily "waf-example".
+          ├── manifest.yml
+          └── addons/
+              ├── waf.yml 
+              └── addons.parameters.yml
+  ```
+
+Copy and paste the following content to the respective files:  
+
+=== "waf.yml"
+  ```yaml
+  #Addon template to add WAF configuration to your App Runner service.
+    
+  Parameters:
+    App:
+      Type: String
+      Description: Your application's name.
+    Env:
+      Type: String
+      Description: The environment name your service, job, or workflow is being deployed to.
+    Name:
+      Type: String
+      Description: The name of the service, job, or workflow being deployed.
+    ServiceARN:
+      Type: String
+      Default: ""
+      Description: The ARN of the service being deployed.
+    
+  Resources:
+    # Configuration of the WAF Web ACL you want to asscoiate with 
+    # your App Runner service.
+    Firewall:
+      Metadata:
+        'aws:copilot:description': 'Associating your App Runner service with your WAF Web ACL'
+      Type: AWS::WAFv2::WebACLAssociation
+      Properties: 
+        ResourceArn: !Sub ${ServiceARN}
+        WebACLArn:  <paste your WAF Web ACL ARN here> # Paste your WAF Web ACL ARN here.
+  ```
+
+=== "addons.parameters.yml"  
+  ```yaml
+  Parameters:
+    ServiceARN: !Ref Service
+  ```
+
+
+### Step 4: Populate your Web ACL ARN in `waf.yml`
+
+Open `waf.yml` and replace `<paste your WAF Web ACL ARN here>` with your the real ARN of your Web ACL resource. For example:   
+   ```yaml
```

Comment:
Why do we need the extra `tab` here 🤔 ? Does it render nicely when you run `make site-local`?

Reply:
Looks like this was a typo as well. Fixed it.